modules/rsyslog/templates/tail-hilary.erb

```diff
@@ -16,7 +16,7 @@ while getopts "n:" opt; do
     case "$opt" in
     n)  lines=$OPTARG
         ;;
-    f)  follow='-f'
+    f)  follow='yes'
```

Comment:
bash has no concept of boolean so using '1' instead of 'yes' makes no difference.